src/ompl/tools/thunder/Thunder.h

```diff
@@ -245,6 +269,11 @@ namespace ompl
 
             bool hybridize_{true};
 
+            std::size_t minSolCount_ {21};
```

Comment:
If we want a chance at hybridize = false to work as intended, minSolCount needs to be 1.
It is used for two things here, determine when we stop all threads and return if hybridize = false, and the number of solutions we start hybridizing after when hybridize = true.
If we care about the second use, we need to decouple the two uses.

Reply:
I set it to one, we can do any further work in a subsequent PR :)